ios/KalturaPlayerViewManager.swift

```diff
@@ -384,6 +403,19 @@ class KalturaPlayerRNView : UIView {
         kalturaPlayer.loadMedia(options: mediaOptions) { error in
             if (error != nil) {
                 print("Error in loadMedia: %@", error!)
+                KalturaPlayerEvents.emitter.sendEvent(withName: "loadMediaFailed", body: [
+                    "code": "ConcurrencyLimitation",
+                    "message": "Concurrency limitation",
+                    "name": "OTTError",
```

Comment:
this shouldnt be hardcoded, you need to get the error from the player and just make sure the object has the same structure like JS layer expects